README.md

```diff
@@ -74,19 +83,19 @@ for _ in 0..<1000 {
 }
 ```
 
-For more models, go to [**tensorflow/swift-models**](https://github.com/tensorflow/swift-models).
+For more models, go to [**tensorflow/swift-models**][swift-models].
 
 ## Development
 
 ### Requirements
 
-* [Swift for TensorFlow toolchain](https://github.com/tensorflow/swift/blob/master/Installation.md).
+* [Swift for TensorFlow toolchain][toolchain].
 * An environment that can run the Swift for TensorFlow toolchains: Linux 18.04 or macOS with Xcode 10.
-* Bazel. This can be installed [manually](https://docs.bazel.build/versions/master/install.html) or with
-[Baselisk](https://github.com/bazelbuild/bazelisk). You will need a version supported by TensorFlow
+* Bazel. This can be installed [manually][bazel] or with
+[Bazelisk][bazelisk]. You will need a version supported by TensorFlow
 (between `_TF_MIN_BAZEL_VERSION` and `_TF_MAX_BAZEL_VERSION` as specified in
-[tensorflow/configure.py](https://github.com/tensorflow/tensorflow/blob/master/configure.py)).
-* Python3 with [numpy](https://numpy.org/).
+[tensorflow/configure.py][configure.py]).
+* Python3 with [numpy][numpy.org].
```

Comment:
This refers to "numpy.org", but the link id is "numpy" so it's not rending correctly.

Reply:
Good catch! Fixed